cocos/ui/UIScale9Sprite.cpp

```diff
@@ -515,18 +426,9 @@ namespace ui {
             return false;
         }
 
-        SpriteFrame *spriteFrame = _scale9Image->getSpriteFrame();
-
-        if (!spriteFrame)
-        {
-            return false;
-        }
-
         Rect rect(textureRect);
         Size size(originalSize);
         
-        _capInsets = capInsets;
```

Comment:
@dabingnn  I found it, because the _capInsets are not needed any more. Only the _capInsetsInternal is used ATM.